apps/src/templates/CertificateShare.jsx

```diff
@@ -1,20 +1,53 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import i18n from '@cdo/locale';
+import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+import {TwoColumnActionBlock} from '@cdo/apps/templates/studioHomepages/TwoColumnActionBlock';
+import styleConstants from '../styleConstants';
 
 export default function CertificateShare(props) {
+  const {announcement} = props;
   return (
-    <a href={props.printUrl}>
-      <img
-        src={props.imageUrl}
-        alt={i18n.certificateForCompletion()}
-        width="100%"
+    <div style={styles.wrapper}>
+      <a href={props.printUrl}>
+        <img
+          src={props.imageUrl}
+          alt={i18n.certificateForCompletion()}
+          width="100%"
+          style={styles.certificate}
+        />
+      </a>
+      <TwoColumnActionBlock
+        isRtl={false}
```

Comment:
It looks like `isRtl` is also available in redux, is there a reason we're also setting it explicitly here?